docs/versioned_docs/version-zenith/user/quickstart/520219-publish.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: /user/quickstart/292472/publish
+hide_table_of_contents: true
+title: "Add functions to publish the application"
+---
+import PartialExperimentalDocs from '../../partials/_experimental.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Add functions to publish the application
+
+At this point, your Dagger module has functions to test and build the application. However, all Dagger SDKs have built-in support to publish container images to remote registries.
```

Comment:
> [...] all Dagger SDKs have built-in support to [...]

That's because it comes from the API, it's not a convenience in the SDKs. So I'd rather say something like "the Dagger API has support" instead.

Reply:
Added